File: beginner/javascript/1929.js
var input = require('fs').readFileSync('stdin', 'utf8');

var linhas = input.trim().split('\n');

for (var i = 0; i < linhas.length; i++) {
    var numeros = linhas[i].split(" ").map(Number);

    for (var j = 0; j < 3; j++) {
        for (var k = j + 1; k < 4; k++) {
            if (numeros[j] > numeros[k]) {
                var temp = numeros[j];
                numeros[j] = numeros[k];
                numeros[k] = temp;
            }
        }
    }

    if (numeros[0] + numeros[1] > numeros[2] || numeros[1] + numeros[2] > numeros[3]) 
        console.log("S");
    else 
        console.log("N");
}

File: beginner/python/1929.py
numeros = list(map(int, input().split()))

for i in range(0, 3): 
    for j in range(i + 1, 4): 
        if numeros[i] > numeros[j]: 
            temp = numeros[i]
            numeros[i] = numeros[j]
            numeros[j] = temp

if numeros[0] + numeros[1] > numeros[2] or numeros[1] + numeros[2] > numeros[3]:
    print("S")
else:
    print("N")
